pages.zh/common/mv.md

```diff
@@ -0,0 +1,28 @@
+# mv
+
+> 移动或重命名文件或目录。
+> 更多信息：<https://www.gnu.org/software/coreutils/mv>.
+
+- 移动文件到任意位置：
+
+`mv {{source}} {{target}}`
+
+- 移动文件到其他目录，并保持文件名不变：
+
+`mv {{source1}} {{source2}} {{source3}} {{target_directory}}`
```

Comment:
```suggestion
`mv {{来源1}} {{来源2}} {{来源3}} {{目标目录}}`
```

Reply:
Thanks for reply, the problems have been resolved.